app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
```

Comment:
1. Coding convention:
+ Nháy đôi hì
+ Sau `stylesheet_link_tag` chỉ 1 space chú hì

2. `SampleApp`
Em tìm hiểu về I18n nhé
Sau đó sử dụng I18n cho `toàn bộ text` của tất cả các page `html` nhé